dataplex/quickstart/src/test/java/dataplex/QuickstartIT.java

```diff
@@ -109,9 +109,7 @@ public void testQuickstart() {
                 "Step 5: Retrieved entry -> projects/%s/locations/%s/entryGroups/%s/entries/%s",
                 PROJECT_ID, LOCATION, ENTRY_GROUP_ID, ENTRY_ID),
             // Step 6 - result from Search
-            String.format(
-                "projects/%s/locations/%s/entryGroups/%s/entries/%s",
-                PROJECT_ID, LOCATION, ENTRY_GROUP_ID, ENTRY_ID),
+            "Entries found in Search:",
```

Comment:
Should there be some Entry ID etc after the colon here?

Reply:
The problem is that Search might result multiple results, so I cannot guarantee the one we are interested in right after the colon. If I just look for path to resource, it will be covered by different log (as it was before the change). 
However, appearance of this log indicates successful call to Search, and point of tests in code samples is to verify call itself, not that underlying back-end works properly - https://screenshot.googleplex.com/4D73RJu93ZyuPsr